### ISSUE TITLE
[BUG] Does not use the VERSION_STATIC property

### ISSUE THREAD

**Issue by csharpfritz** [brandedoutcast/publish-nuget#46]
**Describe the bug**
Does nothing with the version number assigned through VERSION_STATIC

**Failed Action Log URL (Required)**
A URL to the failed action log.

**To Reproduce**
1. Assign value to VERSION_STATIC

**Expected Behavior**
NuGet package should be created with the assigned version

**Environment:**

**Comment by itniels** [comment]
I ran into this issue as well, I have the Action create the version, and would like to set it manually for the NuGet package.

**Comment by Kritner** [comment]
Yeah I'm a bit unclear on how this is supposed to be integrated, or if this should just be more of a boolean flag?

`GitVersioning` just as an example seems to already "come up with" the version itself, and it produces a package file with that version accordingly (assuming doing `GeneratePackageOnBuild`.  I'm tinkering around a bit with this build action right now and running into that problem.

Seems like some examples that show off the use of this feature may be worthwhile, like how it all fits together.  Though based on some of the issues and prs in this repo, perhaps it's been abandoned?
